gpii/node_modules/transformer/src/js/Transformer.js

```diff
@@ -151,4 +151,64 @@ fluid.registerNamespace("gpii.transformer");
         return (60 * (first.hours - second.hours) + first.minutes - second.minutes) < 0;
     };
 
+    /**
+     * New transformation that lets you ignore a path if it's undefined, and supply another value instead.
+     *
+     * The input format for it should be the following:
+     *
+     * "transform": {
+     *     "type": "fluid.transforms.fallbackValue",
+     *     "inputPath": pathToValue,
+     *     "outputPath": outputPathToStoreValue,
+     *     "default": defaultValue
+     * }
+     *
+     * Spec:
+     *  - inputPath: The input path to test for nullity.
+     *  - outputPath: The output path to put the result value from the transformation.
+     *  - default: The default value to put in the "outputPath" in case of this being null.
+     *
+     * @param {Object} transformSpec Transformation specification.
+     * @param {*} transformer The transformer for the transformation.
+     * @return {Object} The output value resulting of the transformation.
+     */
+    fluid.transforms.fallbackValue = function (transformSpec, transformer) {
```

Comment:
This facility already exists in the core under the transform "firstValue" - https://github.com/fluid-project/infusion/blob/master/src/framework/core/js/ModelTransformationTransforms.js#L119